lms/services/lti_grading/_v13.py

```diff
@@ -97,7 +97,7 @@ def sync_grade(  # noqa: PLR0913
         application_instance: ApplicationInstance,
         lis_outcome_service_url: str,
         grade_timestamp: str,
-        user_grading_id: str,
+        lms_user: LMSUser,
```

Comment:
Change the interface of this function, instead of taking directly the user_id, take the whole user.

Doing you allows the V13 version of the method to pick `lti_v13_user_id` while the LTI111 can eventually pick `lti_user_id`